ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcErrorResponse.java

```diff
@@ -69,4 +74,12 @@ public int hashCode() {
   public String toString() {
     return MoreObjects.toStringHelper(this).add("id", id).add("error", error).toString();
   }
+
+  @JsonIgnore
+  public RpcErrorType getErrorType() {
+    return Arrays.stream(RpcErrorType.values())
+        .filter(e -> e.getCode() == error.getCode() && e.getMessage().equals(error.getMessage()))
+        .findFirst()
+        .get();
+  }
```

Comment:
nit: Might be nicer to have an errorType property on JsonRpcError that is ignored for serialization purposes, so we can just call error.getErrorType() instead of this look-up.

Reply:
Updated but I need to do one lookup anyway when I deserialise the response but is better than doing it every time when requested.